packages/fxa-auth-server/test/local/routes/support.js

```diff
@@ -234,6 +234,36 @@ describe('support', () => {
         nock.isDone();
       });
 
+      it('should handle retrying an update user call', async () => {
+        config.subscriptions.enabled = true;
+        nock(`https://${SUBDOMAIN}.zendesk.com`)
+          .post('/api/v2/requests.json')
+          .reply(201, MOCK_CREATE_REPLY);
+        nock(`https://${SUBDOMAIN}.zendesk.com`)
+          .get(`/api/v2/users/${REQUESTER_ID}.json`)
+          .reply(500)
+          .get(`/api/v2/users/${REQUESTER_ID}.json`)
+          .reply(200, MOCK_NEW_SHOW_REPLY);
+        nock(`https://${SUBDOMAIN}.zendesk.com`)
+          .put(`/api/v2/users/${REQUESTER_ID}.json`)
+          .reply(200, MOCK_UPDATE_REPLY);
```

Comment:
We need a failed PUT before this in order to test the retry at https://github.com/mozilla/fxa/pull/3247/files#diff-c01ed3e5dd5108a3df8ed8fc6bed2383R110-R114

Reply:
Yes, I only added one failure to test, mainly as the blocks were otherwise identical.